### PR TITLE
ja: Update around the toppage

### PR DESCRIPTION
**Pull request by aytdm**
Added `homepage/index.md` and updated `lang.json`. If you have a better translation idea about `welcome`, please give me some advice! :pray:  (I think it might be better to use the original...)

### CODE REVIEW

File: ja/lang.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "vue_jobs": "Vue Jobs"
   },
   "text": {
-    "an_error_occurred": "エラーが発生しました",
+    "an_error_occured": "エラーが発生しました",
```
Comment (inouetakuya) [inline ja/lang.json]: en/lang.json をはじめ他の言語でもタイポしているのですが、正しくは `occurred` なので、タイポのほうに合わせるのではなくて、他の言語の箇所もまとめて、正しいほうに修正ってできそうですか？

それほどコストがかからずに修正できそうであれば、ぜひお願いしたいです 🙏

Comment (aytdm) [reply ja/lang.json]: @inouetakuya あ、たしかに`occurred`が正しいですね:bow:
別PRで出すのでこちらからは削除しました。PTALです

**Comment by nh0627** [comment]
Hi! I think changes here from your force-push(https://github.com/nuxt/docs/compare/0330be2f9a36c144a5bd0b194158f64e7fde2073..64330b6812f97815e449aecd3b8f0f7e8e8fb4ad) is not displayed on the Docs after your force-push.

**Comment by aytdm** [comment]
Hi @nh0627,

Thank you for the information! I think It happen due to deploy failure (ref:  [Netlify Deploy log](https://app.netlify.com/sites/nuxt/deploys/5cc32295ea3f18000823a7f1)). Recently, nuxt/docs repository has been sub-moduleed. Latest [nuxtjs.org](https://github.com/nuxt/nuxtjs.org)@76285f3 works fine in local, So I think It will be displayed correctly if it is fixed.

**Comment by inouetakuya** [comment]
I sent the bug report:

- [Deploy to Netlify failed · Issue #187 · nuxt/nuxtjs.org](https://github.com/nuxt/nuxtjs.org/issues/187)

**Comment by aytdm** [comment]
@inouetakuya Thank you so much!!

**Pull request merged**